Makefile

```diff
@@ -452,20 +452,15 @@ CPPCHECK_OPT:=--quiet --force --error-exitcode=-1
 # Ignore '#error'
 CPPCHECK_OPT+=--suppress=preprocessorErrorDirective
 EXTRA_C_SRCS:=$(wildcard uctest/*.c)
-EXTRA_JCLKLIB_SRCS:=$(wildcard jclklib/common/*.c*)
-EXTRA_JCLKLIB_SRCS+=$(wildcard jclklib/proxy/*.c*)
-EXTRA_JCLKLIB_SRCS+=$(wildcard jclklib/client/*.c*)
-EXTRA_JCLKLIB_SRCS+=$(wildcard jclklib/common/*.h*)
-EXTRA_JCLKLIB_SRCS+=$(wildcard jclklib/proxy/*.h*)
-EXTRA_JCLKLIB_SRCS+=$(wildcard jclklib/client/*.h*)
+EXTRA_JCLKLIB_SRCS:=$(wildcard $(JCLKLIB_SRC)/*.cpp)
```

Comment:
Hi peter, i know this is WIP. but just a short comment. 
Does this work ? It seems to me we have nested folders . 

JCLKLIB_SRC:=jclklib
SRCS_JCLKLIB:=$(wildcard $(JCLKLIB_SRC)/*.cpp)

EXTRA_SRCS:=$(wildcard $(foreach n,$(JCLKLIB_SRC)/client  $(JCLKLIB_SRC)/common $(JCLKLIB_SRC)/proxy,$n/*.cpp $n/*.h)) <-- using something like this might pass.

Reply:
yeah Azura, the CI test is passing with EXTRA_JCLKLIB_SRCS:=$(wildcard $(JCLKLIB_SRC)/*.cpp).